src/editor.js

```diff
@@ -104,6 +104,7 @@ export class OdooEditor extends EventTarget {
         this._domListeners = [];
 
         this.resetHistory();
+        this._historyActive = true;
```

Comment:
I would call it `_historySkipStep` because history active seems more generic and therefore could be understood too broadly. The history can still be used (undo/redo/collab/...) and the historyObserver continues to collect mutations, it is just the steps being skipped.

Reply:
I agree that the name is not ideal but I don't think `_historySkipStep` does the trick either because it makes me think it's just going to skip _one_ step, while in fact `historyStep` is completely deactivated and will _have to_ be reactivated afterwards.

Reply:
`historySkipSteps` ?

Reply:
HistoryStepsActive ?

Reply:
`historyStepsActive` is good I think. Then the methods become `historyPauseSteps` and `historyUnpauseSteps`.